oauth2-authorization-server/src/test/java/org/springframework/security/oauth2/server/authorization/config/ProviderSettingsTests.java

```diff
@@ -41,6 +44,74 @@ public void buildWhenDefaultThenDefaultsAreSet() {
 		assertThat(providerSettings.getOidcUserInfoEndpoint()).isEqualTo("/userinfo");
 	}
 
+	@Test
+	public void validateIssuerWithRfc8414() {
```

Comment:
Please move this test to `OidcTests` - see similar tests `loadContextWhenIssuerNotValidUrlThenThrowException` and `loadContextWhenIssuerNotValidUriThenThrowException`